### ISSUE TITLE
An easy way to give a name to an unnamed <value> column

### ISSUE THREAD

**Issue by nespera** [nushell/nushell#1698]
**Is your feature request related to a problem? Please describe.**
Various commands use the names of the columns as input and don't work with an unnamed <value> column. For example, if I do this:
`ls | pick name type | each { to-url }`

I get a result like this:
```
 #  │ <value> 
────┼───────────────────────────────────────────
  0 │ "name=bin&type=Dir" 
  1 │ "name=data&type=Dir" 
```

If I then want to use `format` to build a full URL there seems to be no way to refer to that unnamed column (or at least I can't find one). I could only do this to get it to work:
`ls | pick name type | each { to-url } | split-column "|" q | format "http://localhost:8080/thing?{q}"`

In other words, doing a split column on a character that doesn't appear in order to name the column.

**Describe the solution you'd like**
A command to name the unnamed column. Maybe `as-col`:
`ls | pick name type | each { to-url } | as-col q | format "http://localhost:8080/thing?{q}"`

**Describe alternatives you've considered**
Maybe there is a way to do this already and I just can't find it!

**Additional context**

**Comment by jzaefferer** [comment]
Related to #903

Btw. I couldn't find any mention of `<value>` in the book, as if that doesn't exist at all.

Not even the docs for the get command has that anywhere: https://www.nushell.sh/commands/get.html Even though its really easy to produce a list with `<value>` using `get`, like `ls | get name`

**Comment by nespera** [comment]
An alternative I just thought of would be to have a default column name like `value` for commands that currently yield `<value>`.

**Comment by jonathandturner** [comment]
One approach you could use is `wrap <column name>` to give the value column a name. That said, I agree it'd be nice to be able to use a shorthand and treat the value column the same.

**Comment by nespera** [comment]
Ah yes, wrap seems to work. I didn't know about that. Maybe I should use that as a spur to write some documentation for that!

**Comment by thegedge** [comment]
`wrap` is definitely the solution here. I'm going to close this issue, but I think there's room for improvement here, either in error messaging or how pipelines function with "lists" (make them more seamless with tables).
